en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-300.md

```diff
@@ -1710,7 +1710,7 @@ Follow the instructions below to move all the existing API Manager configuration
 
     1.  Download the identity component migration resources and unzip it in a local directory.
 
-         Navigate to the [latest release tag](https://github.com/wso2-extensions/identity-migration-resources/releases/latest) and download the `wso2is-migration-x.x.x.zip` under Assets.
+         Navigate to the [latest release tag](https://github.com/wso2-extensions/identity-migration-resources/releases/latest) and download the `wso2is-migration-x.x.x.zip` under Assets. We ll refer this downloaded and extracted directory as `<IS_MIGRATION_TOOL_HOME>`. 
```

Comment:
Shall we change "We ll refer this downloaded and extracted directory as `<IS_MIGRATION_TOOL_HOME>`." to

Let's refer to this directory that you downloaded and extracted as `<IS_MIGRATION_TOOL_HOME>`.

Reply:
Sure, Please go ahead and make the changes